ts/parts/Axis.ts

```diff
@@ -319,6 +322,7 @@ declare global {
             public height: number;
             public horiz?: boolean;
             public isBroken?: boolean;
+            public isCircular?: (undefined|boolean);
```

Comment:
I think this should be defined in RadialAxis, not in Axis? The same for `clipCrosshair `. Perhaps I'm wrong, @bre1470 ?

Reply:
Got rid of the `clipCrosshair` as it isn't necessary any more (crosshair's length is calculated in the `getPlotLinePath` function) and as it seems the `isCircular`  it was already defined in the RadialAxis part.